README.md

```diff
@@ -82,6 +82,8 @@ Or install it yourself as:
 Create a command instance and then run some commands:
 
 ```ruby
+require 'tty/command'
```

Comment:
Thanks! However, though it works I would normally require this and other tty gems with a dash like `tty-command` and this would be my preference. Do you mind changing?

Reply:
I don't understand, `require 'tty/command'` and `require 'tty-command'` are going to require different files, no? Can't only one of them be correct?

Reply:
It's not necessarily about correctness but the usual hard thing of naming things. [rubygems.org](http://guides.rubygems.org/name-your-gem/) has recommendation that if a gem uses dash then the gem user should be able to require it with forward slash. However, a lot of people new to Ruby are not aware of the convention and require gems by their actual name. Therefore, I make sure that both are valid ways for loading the library. Neither is correct, but I have a slight preference to make it simple and stick with the actual name in the docs.

Reply:
The thing that confused me (I'm quite new to ruby) was `include TTY::Color` from https://github.com/piotrmurach/tty-color#usage and the fact that this repo's README didn't mention how you can "import" this gem. So I just found out `require 'tty/command'` in an older issue after digging a while.

Reply:
@revolter I totally understand that! The documentation is really important to me and I wish to make it approachable to any level Ruby dev. However, due to my years in Ruby world I have kind of forgotten what things you need to know to get started. Sorry!